### ISSUE TITLE
Fresh install of OpenShot form Ubunut Software didn't start in Ubuntu 22.04 LTS 

### ISSUE THREAD

**Issue by Aravinth-Earth** [OpenShot/openshot-qt#4838]
**Describe the bug:**
When i tried to open the app from apps list or via terminal the application is not starting at all. The detailed outcomes listed below.

## Approach 1 - GUI way
**Steps to reproduce the behavior:**
1. Click on 'Show Applications' from Ubuntu Dock'
2. Click on 'OpenShot

**Expected behavior:**
The successful launch of the application

**Actual behavior:**
Nothing is happening or showing no error pop-ups or similar

## Approach 2 - CLI way
**Steps to reproduce the behavior:**
1. Launch terminal
2. type 'openshot-qt' and hit Enter

**Expected behavior:**
The successful launch of the application

**Actual behavior:**
Just receiving the below list of error(s)
```
user@pc:~$ openshot-qt 
Loaded modules from installed directory: /usr/lib/python3/dist-packages/openshot_qt
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
         app:INFO ------------------------------------------------
         app:INFO             Tue Jun 28 15:04:03 2022            
         app:INFO               Starting new session              
         app:INFO ------------------------------------------------
         app:INFO             OpenShot (version 2.5.1)            
         app:INFO ------------------------------------------------
         app:INFO openshot-qt version: 2.5.1
         app:INFO libopenshot version: 0.2.5
         app:INFO platform: Linux-5.15.0-40-generic-x86_64-with-glibc2.35
         app:INFO processor: x86_64
         app:INFO machine: x86_64
         app:INFO python version: 3.10.4
         app:INFO qt5 version: 5.15.3
         app:INFO pyqt5 version: 5.15.6
     metrics:ERROR Error determining OS version in metrics.py
    language:INFO Qt Detected Languages: ['en-GB', 'en']
    language:INFO LANG Environment Variable: en_GB.UTF-8
    language:INFO LOCALE Environment Variable: 
    language:INFO OpenShot Preference Language: Default
project_data:INFO Setting default profile to HD 720p 30 fps
ZmqLogger::Connection - Error binding to tcp://*:5556. Switching to an available port.
ZmqLogger::Connection - Error binding to tcp://*:5556. Switching to an available port.
         app:INFO Setting font to Cantarell
logger_libopenshot:INFO Connecting to libopenshot with debug port: 5556
         app:INFO Setting custom dark theme
QMainWindow::addDockWidget: invalid 'area' argument
     ui_util:INFO Initializing UI for MainWindow
  exceptions:ERROR Unhandled Exception
Traceback (most recent call last):
  File "/usr/bin/openshot-qt", line 33, in <module>
    sys.exit(load_entry_point('openshot-qt==2.5.1', 'gui_scripts', 'openshot-qt')())
  File "/usr/lib/python3/dist-packages/openshot_qt/launch.py", line 97, in main
    app = OpenShotApp(argv)
  File "/usr/lib/python3/dist-packages/openshot_qt/classes/app.py", line 219, in __init__
    self.window = MainWindow(mode)
  File "/usr/lib/python3/dist-packages/openshot_qt/windows/main_window.py", line 2521, in __init__
    self.timeline = TimelineWebView(self)
  File "/usr/lib/python3/dist-packages/openshot_qt/windows/views/timeline_webview.py", line 3000, in __init__
    self.cache_renderer.setInterval(0.5 * 1000)
TypeError: setInterval(self, int): argument 1 has unexpected type 'float'
     version:INFO Found current version: {"trans_rate_stable": 0.01, "trans_rate_unstable": 0.001, "error_rate_unstable": 0.05, "openshot_version": "2.6.1", "error_rate_stable": 0.16}
```

**System Details:**
- OpenShot Version 2.5.1
- Operating System / Distro: Ubuntu Desktop 22.04 LTS

**Log Files:**
Not Applicable

**Exception / Stacktrace:**
Added above in Actual outcome under Approach 2 - CLI way

**Screenshots:** (Optional)
Not Applicable

**Comment by JacksonRG** [comment]
Hi, 2.6.1 is the current release version we're supporting.

If your software center doesn't show that, you can download it [here](https://www.openshot.org/download/). And if that still has an issue, you can also find our daily builds, which have more recent fixes. (Though may be somewhat less stable, as we're actively improving that towards our next release.)

**Comment by c-git** [comment]
# TLDR 

The version installed from the software center used to work and is no longer working. Last used in May 23, 2022 and it was working. Hadn't used it since and it's not working anymore when I tried it today. But the AppImage works.

## Brief note about AppImages

The current stable AppImage `OpenShot-v2.6.1-x86_64.AppImage` has an [issue](https://github.com/OpenShot/openshot-qt/pull/4787#issuecomment-1160941558) that causes none of the tracks to be displayed.
Had to use the daily `OpenShot-v2.6.1-dev-daily-10349-5aa9487e-46255e46-x86_64.AppImage`


# Details

I think I'm having the same problem. I tried to reinstall but maybe I did it wrong. I used the instructions I found [here](https://github.com/OpenShot/openshot-qt/issues/3772#issuecomment-757013441).

Actual commands used to attempt reinstall.

```bash
sudo apt purge openshot-qt libopenshot19 libopenshot-audio7 python3-openshot 
sudo apt install openshot-qt
```

When I try to run it I get something slightly different from the OP.

```
$openshot-qt 
Loaded modules from installed directory: /usr/lib/python3/dist-packages/openshot_qt
         app:INFO ------------------------------------------------
         app:INFO             Sun Nov 20 21:15:50 2022            
         app:INFO               Starting new session              
         app:INFO ------------------------------------------------
         app:INFO             OpenShot (version 2.5.1)            
         app:INFO ------------------------------------------------
         app:INFO openshot-qt version: 2.5.1
         app:INFO libopenshot version: 0.2.5
         app:INFO platform: Linux-5.15.0-53-generic-x86_64-with-glibc2.35
         app:INFO processor: x86_64
         app:INFO machine: x86_64
         app:INFO python version: 3.10.6
         app:INFO qt5 version: 5.15.3
         app:INFO pyqt5 version: 5.15.6
     metrics:ERROR Error determining OS version in metrics.py
    language:INFO Qt Detected Languages: ['en-CA', 'en']
    language:INFO LANG Environment Variable: en_CA.UTF-8
    language:INFO LOCALE Environment Variable: 
    language:INFO OpenShot Preference Language: Default
project_data:INFO Setting default profile to HD 720p 30 fps
         app:INFO Setting font to Cantarell
         app:INFO Setting custom dark theme
logger_libopenshot:INFO Connecting to libopenshot with debug port: 5556
QMainWindow::addDockWidget: invalid 'area' argument
     ui_util:INFO Initializing UI for MainWindow
  exceptions:ERROR Unhandled Exception
Traceback (most recent call last):
  File "/usr/bin/openshot-qt", line 33, in <module>
    sys.exit(load_entry_point('openshot-qt==2.5.1', 'gui_scripts', 'openshot-qt')())
  File "/usr/lib/python3/dist-packages/openshot_qt/launch.py", line 97, in main
    app = OpenShotApp(argv)
  File "/usr/lib/python3/dist-packages/openshot_qt/classes/app.py", line 219, in __init__
    self.window = MainWindow(mode)
  File "/usr/lib/python3/dist-packages/openshot_qt/windows/main_window.py", line 2521, in __init__
    self.timeline = TimelineWebView(self)
  File "/usr/lib/python3/dist-packages/openshot_qt/windows/views/timeline_webview.py", line 3000, in __init__
    self.cache_renderer.setInterval(0.5 * 1000)
TypeError: setInterval(self, int): argument 1 has unexpected type 'float'
     version:INFO Found current version: {"trans_rate_stable": 0.01, "trans_rate_unstable": 0.001, "openshot_version": "2.6.1", "error_rate_unstable": 0.05, "error_rate_stable": 0.16}
```

Just providing this in case it's useful.

**Comment by Colorjet3** [comment]
Openshot 2.6.1-Release-Candidate is very buggy and one of the known issues is the fact that the timeline never displays.  To avoid this issue and others, please go to openshot.org/download/#daily and download the latest DEV Daily Build (AppImage).  If you want to actually install it then follow the instructions here:  https://www.openshot.org/ppa/.  Please execute the Daily PPA commands and not the Stable PPA commands.

A production release with many bug fixes and enhancements is just around the corner.
